server/services/clean-up.js

```diff
@@ -101,24 +103,48 @@ const createStaleOpenParticipantsTable = async () => {
       AND ps.current = 'true'
       AND ps.created_at < (NOW() - interval '6 month')
     WHERE p.body->>'interested' = 'yes'
-    -- If anyone has a status in the past 6 months, exclude them
+    -- If anyone has a non-rejected status in the past 6 months or is has a current non-rejected status, exclude them
     AND p.id NOT IN (
       SELECT DISTINCT ON (ps.participant_id)
       ps.participant_id
       FROM participants_status ps
-      WHERE ps.created_at > (NOW() - interval '6 month')
-      AND ps.current = true
+      WHERE ps.status NOT IN ('rejected', 'reject_ack')
+      AND (
+        ps.created_at > (NOW() - interval '6 month')
+        OR ps.current = true
+      )
       GROUP BY ps.participant_id
     )
     GROUP BY p.id;
   `;
   await dbClient.db.query(pushRejectedParticipants);
+
+  // Anyone who was affected by the stale, in progress script will have only CURRENT=FALSE statuses
```

Comment:
Why do they need to be addressed by this script? Just to set `interested = 'withdrawn'`?